canvas_sdk/v1/data/questionnaire.py

```diff
@@ -0,0 +1,165 @@
+
+
+class Question(models.Model):
+    """Question."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_question_001"
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    status = models.CharField()
+    name = models.CharField()
+    response_option_set = models.ForeignKey(
+        ResponseOptionSet, on_delete=models.CASCADE, related_name="questions"
+    )
+    acknowledge_only = models.BooleanField()
+    show_prologue = models.BooleanField()
+    code_system = models.CharField()
+    code = models.CharField()
+
+
+class Questionnaire(models.Model):
+    """Questionnaire."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_questionnaire_001"
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    created = models.DateTimeField()
+    modified = models.DateTimeField()
+    status = models.CharField()
+    # graph_header
+    # graph_nodes
+    # graph_edges
+    name = models.CharField()
+    expected_completion_time = models.FloatField()
+    can_originate_in_charting = models.BooleanField()
+    use_case_in_charting = models.CharField()
+    scoring_function_name = models.TextField()
+    scoring_code_system = models.CharField()
+    scoring_code = models.CharField()
+    code_system = models.CharField()
+    code = models.CharField()
+    search_tags = models.CharField()
+    # TODO: Need to bring over questionnaire question map?
+    # questions = models.ManyToManyField(Question, through="QuestionnaireQuestionMap")
+    use_in_shx = models.BooleanField()
+    carry_forward = models.TextField()
+
+
+class Interview(models.Model):
+    """Interview."""
+
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_interview_001"
+
+    objects = CommittableModelManager()
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
+    deleted = models.BooleanField()
+    committer = models.ForeignKey(CanvasUser, on_delete=models.DO_NOTHING)
+    entered_in_error = models.ForeignKey(CanvasUser, on_delete=models.DO_NOTHING)
+    status = models.CharField()
+    # graph_header
+    # graph_nodes
+    # graph_edges
+    # skipped_questions
+    name = models.CharField()
+    language_id = models.BigIntegerField()
+    use_case_in_charting = models.CharField()
+    patient = models.ForeignKey(Patient, on_delete=models.CASCADE, related_name="interviews")
+    note_id = models.BigIntegerField()
+    appointment_id = models.BigIntegerField()
+    # TODO: Need to bring over interview quesiton map?
+    # questionnaires = models.ManyToManyField(Questionnaire, through="InterviewQuestionnaireMap")
+    # data = models.JSONField()
+    progress_status = models.CharField()
```

Comment:
This, among other fields in models we've already created, is a choice field. We haven't brought the choice enums over, but perhaps we should because it would make filtering easier for users.

Reply:
This field might not be used at all, but your broader point is valid. I agree that we should bring over the constants to help filtering.

Reply:
Yeah that's what I figured. Better to do this all in one shot as a discrete task.